key/key.go

```diff
@@ -22,15 +27,11 @@ var (
 
 // Key defines methods for key manager interface.
 type Key interface {
-	// P returns all formatted P-Chain addresses.
-	P(string) (string, error)
-	// C returns the C-Chain address in Ethereum format
-	C() string
 	// Addresses returns the all raw ids.ShortID address.
 	Addresses() []ids.ShortID
```

Comment:
Currently Addresses returns an array of one element, why so and will there be a case when there is multiple addresses?